package.json

```diff
@@ -90,6 +92,6 @@
     "watch:stubs": "chokidar \"./solutions/stubs/src/**/*\" \"./solutions/commons/**/*\" -c \"npm run build-local:stubs\"",
     "run:stubs": "([ -n \"$RUN_ALL\" ] || sh solutions/localstack/provision.sh) && npm run build-local:stubs && concurrently 'npm run watch:stubs' 'cd solutions/stubs && node --enable-source-maps --watch --env-file=$([ \"$IS_INTEGRATION_TEST\" = \"1\" ] && echo \".env.integration-tests\" || echo \".env\") ./dist/local.js' --kill-others",
     "run:all": "sh solutions/localstack/provision.sh && RUN_ALL=1 concurrently 'npm run run:frontend' 'npm run run:stubs' --kill-others",
-    "config:validate": "for f in solutions/config/*.json; do (jsonschema validate solutions/config/schema/config.json $f) || exit 1; done"
+    "config:validate": "for f in solutions/config/*.json; do (check-jsonschema --schemafile solutions/config/schema/config.json $f) || exit 1; done"
```

Comment:
Hi @peterfajemisincabinetoffice , @danacotoran - Wondering why we swapped out `jsonschema` for `check-jsonschema` - how is it installed - `jsonschema` is installed via `brewfile` btw?